### PR TITLE
refs task #26427 manage tags

### PR DESCRIPTION
**Pull request by vuchuc789**
## Related Tickets:
- [#26427](https://edu-redmine.sun-asterisk.vn/issues/26427)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by vuchuc789** [comment]
ready

**Comment by vuchuc781999** [comment]
ready

**Comment by vuchuc789** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -41,6 +42,11 @@
     ]);
 
     Route::get('searched', 'SearchController@searchedQuestions')->name('search');
+
+    Route::group(['prefix' => 'tag'], function () {
+        Route::get('/list', 'TagController@list')->name('tags');
+        Route::get('/list_questions/{tag}', 'TagController@listQuestions')->name('tag_question');
```
Comment (huyenttm-0901) [inline routes/web.php]:
```suggestion
        Route::get('/list-questions/{tag}', 'TagController@listQuestions')->name('tag_question');
```

Comment (vuchuc789) [reply routes/web.php]: Em đã sửa ạ. 😄

**Comment by vuchuc789** [comment]
ready

**Pull request merged**